rmf_fleet_msgs/msg/RobotMode.msg

```diff
@@ -13,5 +13,9 @@ uint32 MODE_DOCKING=7
 uint32 MODE_ADAPTER_ERROR=8
 
 uint32 MODE_CLEANING=9
+uint32 MODE_PERFORMING_ACTION=10
+uint32 MODE_ACTION_COMPLETED=11
 
 uint64 mode_request_id
+
+string performing_action
```

Comment:
Nit: Can we add a docstring explaining what this field is for?

Reply:
[0251a9c](https://github.com/open-rmf/rmf_internal_msgs/pull/70/commits/0251a9c9edde99facefca0157959b7f10cd51521)